student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
```

Comment:
- [ ] You are using the colon notation for `parent_permission`. This means the way of assigning it in the `super()` method would be different.

  ```
  super(age, name, parent_permission: parent_permission)
  ```